packages/web-forms/src/components/widgets/IconSVG.vue

```diff
@@ -0,0 +1,125 @@
+<script setup lang="ts">
+import { computed } from 'vue';
+import {
+	mdiAlert,
+	mdiAlertCircleOutline,
+	mdiCamera,
+	mdiCheck,
+	mdiChevronDown,
+	mdiChevronUp,
+	mdiClose,
+	mdiDownload,
+	mdiDragVertical,
+	mdiEye,
+	mdiFileDocumentOutline,
+	mdiImage,
+	mdiMapMarkerOutline,
+	mdiMenu,
+	mdiPrinter,
+	mdiRefresh,
+	mdiUnfoldMoreHorizontal,
+	mdiUpload,
+	mdiWeb,
+} from '@mdi/js';
+
+// Find icons here: https://pictogrammers.com/library/mdi/
+const iconMap: Record<string, string> = {
```

Comment:
We don't have that many icons. If it grows a lot eventually, we will extract it to a file; for now, it's good to have it in one place.